Documentation/Functions/Stdwrap.rst

```diff
@@ -717,6 +722,87 @@ strftime
         <https://www.php.net/gmstrftime>`_ will be used instead of
         `strftime() <https://www.php.net/strftime>`_.
 
+    ..  note::
+        You should consider using the more flexible function
+        :t3-function-stdwrap:`formattedDate`.
+
+formattedDate
+~~~~~~~~~~~~~
+
+..  versionadded:: 12.3
+
+..  t3-function-stdwrap:: formattedDate
+
+    :Data type: :t3-data-type:`string`
+
+    The function renders date and time based on formats/patterns defined by
+    the International Components for Unicode standard (ICU). ICU-based date and
+    time formatting is much more flexible in rendering as
+    :t3-function-stdwrap:`date` or :t3-function-stdwrap:`strftime`, as it ships
+    with default patterns for date and time based on the given locale (given
+    examples for locale `en-US` and timezone `America/Los_Angeles`):
+
+    *   `FULL`, for example: `Friday, March 17, 2023 at 3:00:00 AM Pacific Daylight Time`
+    *   `LONG`, for example: `March 17, 2023 at 3:00:00 AM PDT`
+    *   `MEDIUM`, for example: `Mar 17, 2023, 3:00:00 AM`
+    *   `SHORT`, for example: `3/17/23, 3:00 AM`
+
+    TYPO3 also adds prepared custom patterns:
+
+    *   `FULLDATE`, for example: `Friday, March 17, 2023`
+    *   `FULLTIME`, for example: `3:00:00 AM Pacific Daylight Time`
+    *   `LONGDATE`, for example: `March 17, 2023`
+    *   `LONGTIME`, for example: `3:00:00 AM PDT`
+    *   `MEDIUMDATE`, for example: `Mar 17, 2023`
+    *   `MEDIUMTIME`, for example: `3:00:00 AM`
+    *   `SHORTDATE`, for example: `3/17/23`
+    *   `SHORTTIME`, for example: `3:00 AM`
+
+    ..  note::
+        You can specify an own pattern to suit your requirements, for example:
```

Comment:
```suggestion
        You can specify your own pattern to suit your requirements, for example:
```

Reply:
Double "your" - would like to avoid that.

Reply:
Specify your own pattern ...?

Reply:
specify **your** own pattern to suit **your** requirements

Reply:
Well, whatever